2200078/파리퇴치.py

```diff
@@ -0,0 +1,23 @@
+def get_x(arr, v):
+    s = sum(sum(arr[i][j] for i in range(v[0], v[0]+M) if i in range(N)) for j in range(v[1], v[1]+M) if j in range(N))
```

Comment:
깔끔한 풀이네요! 👍
```suggestion
    s = sum(sum(arr[i][j] for i in range(v[0], v[0]+M) if i < N) for j in range(v[1], v[1]+M) if j < N)
```
`<`를 활용하여 표시하면 더욱 좋을 것 같습니다 😄